00new_packages.autobuild

```diff
@@ -32,4 +32,9 @@ ruby_package 'tools/rest_api' do |pkg|
 end
 
 cmake_package "gui/osg_qt4"
+if (Autoproj.workspace.operating_system[1] & ['18.04']).any? && !(Autoproj.workspace.operating_system[0] & ['gentoo']).any?
+    #force the source package even if an osdep could be found
+    #this may be needed for ubuntu 19.04 and below, but definitely for 18.04, and definitely not for gentoo
+    Autoproj.add_osdeps_overrides 'gui/osg_qt4', :package => 'gui/osg_qt4', :force => true
+end
```

Comment:
The now-preferred way to do this is to mark the package as nonexistent in the osdeps definition, e.g.

~~~
gui/osg_qt4:
  ubuntu:
     "18.04": nonexistent
  gentoo: nonexistent
~~~

It will force autoproj to consider the source package instead.

Reply:
Sorry about the mess, will pull the gentoo osdeps that are not relevant to qt out and into a PR against master, then look at merging master into feature/qt5 and have the rest of the changes there(minus the transformer change, and with the change for osg_qt4).